utils/rpms/daos.spec

```diff
@@ -216,8 +219,12 @@ scons %{?_smp_mflags}                 \
       USE_INSTALLED=all               \
       CONF_DIR=%{conf_dir}            \
       PREFIX=%{_prefix}               \
-      %{?scons_args}
+      %{?scons_args}                  \
+      %{?compiler_args}
 
+%if ("%{?compiler_args}" == "COMPILER=covc")
+cp test.cov %{buildroot}/usr/lib/daos/TESTING/ftest/
```

Comment:
Is there something in this file?  What is in it?

Reply:
the instrumented code for tests